test/Cronofy.Test/CronofyAccountClientTests/GetEvents.cs

```diff
@@ -46,6 +46,7 @@ public void CanGetEventWithoutLocation()
       ""end"": ""2014-09-08"",
       ""deleted"": false,
       ""recurring"": true,
+      ""series_identifier"": ""identifier"",
```

Comment:
It'd be worth having one of these test cases look like a regular non-recurring event still to cover our bases, so having the value be `null` or omitted (whichever the real API does) and presumably asserting the resulting event object also has a `null` value.